analysis/behavioral_analysis/cannabis_table.ipynb

```diff
@@ -0,0 +1,228 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea95a452",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from collections import defaultdict\n",
+    "from tableone import TableOne\n",
+    "import glob\n",
+    "import re\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d9cc49ce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def rm_CUD_baseline(subs_all):\n",
+    "    \n",
+    "    #subs to be excluded (only MM)\n",
```

Comment:
```suggestion
    "    #subs to be excluded (only MM)\n",
    "    #these are excluded from the study because they had cannabis use disorder at baseline (exclusion criterium) 
```

Reply:
```suggestion
    "    #subs to be excluded (only MM) because they had cannabis use disorder at baseline (exclusion criterium)\n",
```